packages/synapse-react-client/src/components/GenericCard/GenericCard.tsx

```diff
@@ -416,6 +425,20 @@ class _GenericCard extends Component<GenericCardPropsInternal> {
     )
     const values: string[][] = []
     const { secondaryLabels = [] } = genericCardSchemaDefined
+    genericCardSchemaDefined.customSecondaryLabelConfig = {
```

Comment:
I would expect customSecondaryLabelConfig to be set in the portal config, not in the Generic Card itself (so anyone using a Generic Card can add a custom hardcoded value, whose visibility logic is custom)